tests/console/sshd.pm

```diff
@@ -28,7 +28,7 @@ sub run {
     select_console 'root-console';
 
     if (is_upgrade && check_var('ORIGIN_SYSTEM_VERSION', '11-SP4')) {
-        record_soft_failure 'bsc#1090178: SuSEfirewall2 service is not available after upgrade from SLES11 SP4 to SLES15';
+        record_info("SuSEfirewall2 not available", "SuSEfirewall2 service is not available after upgrade from SLES11 SP4 to SLES15");
```

Comment:
Isn't the firewalld available? I wonder if it makes sense to stop it in case it's there. Also, I guess it would be nice to introduce check that it's upgrade to SLE15 and not to SLE12, for example. I suppose VERSION variable will be set to 15, so `is_sle` can be used than. WDYT?

Reply:
Please see https://bugzilla.suse.com/show_bug.cgi?id=1090178#c15, the ticket is to remove the workaround for the SuSEfirewall2 service is not available won't be fixed. Since the SLES12 testcases doesn't use variable PATCH=1 so ORIGIN_SYSTEM_VERSION won't be set, so this only affect the migration to SLE15 test.

Reply:
It's ok in general to replace to soft-fail with a record_info. Still I would reference the bug as it has helpful information to understand the behaviour. Please provide the bug ref still. Also, no need for a verification run but the the first string must be very short to fit into the info-box. Maybe try it out by running a very simple fake openQA test that just sets the record_info and nothing else and can execute within seconds.

Reply:
Thanks Okurz, I verified this by adding the line to module  boot_to_desktop. The String can be shown correctly.
http://openqa-apac1.suse.de/tests/1557#step/boot_to_desktop/1